### PR TITLE
Add CI, and kebabify.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# mcp2518fd
+# MCP2518FD
```
Comment (floitsch) [inline README.md:1]: Changed by hand.

File: examples/spi.toit [hunk 2]
```diff
@@ -24,7 +24,7 @@ main:
   driver.configure
 
   task::
-    INT.config --input
+    INT.configure --input
```
Comment (floitsch) [inline examples/spi.toit:27]: Changed by hand.

**Comment by floitsch** [comment]
Added the .github directory, and kebabified.
2 more changes that I have marked with a comment.

**Review by kasperl — APPROVED** [review]
Looks good to me.

**Pull request merged**